### PR TITLE
Fix mapper name in score panel

### LINKED ISSUE #8317: Mapper name replaced by player name in results screen
**Describe the bug:result panel typo** 

**Screenshots or videos showing encountered issue** 

this mapped by beary605
image

result screen say mapped by tladudtn7598
i think change played by tladudtn7598(i not mapped this map)
image

**osu!lazer version:2020.317.0**

### PR DESCRIPTION
**Pull request by recapitalverb**
Fixes #8317

### CODE REVIEW

File: osu.Game/Screens/Ranking/Expanded/ExpandedPanelMiddleContent.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ private void load(Bindable<WorkingBeatmap> working)
         {
             var beatmap = working.Value.BeatmapInfo;
             var metadata = beatmap.Metadata;
+            var creator = beatmap.Metadata.Author?.Username;
```
Comment (bdach) [inline osu.Game/Screens/Ranking/Expanded/ExpandedPanelMiddleContent.cs]: can use `metadata`

**Comment by bdach** [comment]
No tests?

**Comment by recapitalverb** [comment]
(the tests I have added are a bit hacky imo, the beatmap's author reference will stay null until restart unless all steps are run)

**Comment by bdach** [comment]
@recapitalverb I rewrote the tests to use what I consider a cleaner approach - please have a look and let me know if something is unclear in the altered implementation.

**Pull request merged**